pkgs/tools/text/mmdoc/default.nix

```diff
@@ -0,0 +1,67 @@
+{stdenv,
+fetchFromGitHub,
+makeWrapper,
+
+cmark-gfm,
+coreutils,
+findutils,
+gnugrep,
+gnused,
+jq,
+} :
+
+let
+
+  runtimePath = stdenv.lib.makeBinPath [
+    cmark-gfm
+    coreutils
+    findutils
+    gnugrep
+    gnused
+    jq
+  ];
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "mmdoc";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner  = "ryantm";
+    repo   = "mmdoc";
+    rev    = version;
+    hash = "sha256-p6ndIvmY2gkqDpGGv21U7POaS5xKJLn4Zz4jYiqJ2WA=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+    mkdir -p "$out/lib/mmdoc"
+    install "$src/bin/mmdoc" "$out/bin"
+    install "$src/lib/mmdoc/fuse.basic.min.js" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/highlight.pack.js" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/minimal.css" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/mono-blue.css" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/search.js" "$out/lib/mmdoc"
+    patchShebangs $out/bin
```

Comment:
I think stdenv does this in fixup phase?

Reply:
It doesn't work if I don't do that though.

Reply:
That's because you are overriding `fixupPhase`. See https://github.com/jtojnar/nixpkgs-hammering/blob/e414d83e8fbbc920674ee8ae239395704f3b5d54/explanations/fixup-phase.md

Reply:
Fixed all nixpkgs-hammering issues!